lib/std/os.zig

```diff
@@ -4983,11 +4983,11 @@ pub fn sendmsg(
     /// The file descriptor of the sending socket.
     sockfd: socket_t,
     /// Message header and iovecs
-    msg: msghdr_const,
+    msg: std.x.os.Socket.Message,
```

Comment:
I thought the plan was to make an experimental namespace, to try out some new ideas, but this is reaching into the `std.os` namespace and making breaking changes.

Reply:
The reason for replacing `msghdr` and `msghdr_const` in place of `std.x.os.Socket.Message` is that `std.x.os.Socket.Message` defines `msghdr` and `msghdr_const` once for the 16 targets in CI rather than 16 different times in different platform-specific files.

Though, it definitely is a breaking change in hopes of cleaning up some of the OS bits in the current Zig standard library. I'm happy to revert and add back the `msghdr` and `msghdr_const` bits, or another option would be to move `std.x.os.Socket.Message` into std.os.bits since the definition comes straight out from findings in musl/glibc.

What do you think?